### PR TITLE
MF-1087 - Remove WebSocket adapter

### LINKED ISSUE #1087: Remove WebSocket adapter
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
[WebSocket adapter](https://github.com/mainflux/mainflux/tree/master/ws) was built to enable WebSocket protocol before the WS enablement of the MQTT borker.

Today, with mProxy and prowerful scalable MQTT broker there is a very good support of [MQTT-over-WS](https://www.hivemq.com/blog/mqtt-essentials-special-mqtt-over-websockets/). WS is mostly used for web and mobile apps and there are [good front-end libraries](https://github.com/mqttjs/MQTT.js/) today to support MQTT-over-WS.

As an additional benefit, MQTT-over-WS provides important features like QoS, LWT, retnetion, etc.

Because of all this, and with intention to simplify the system, classic WS adapter becomes reundant and should be removed. All aplications should use MQTT-over-WS instead.

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
Must-have

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

Resolves #1087

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

**Comment by drasko** [comment]
I approved this one.

However - one thing to check: NginX conf, does it use WS only over MQTT?

Additionally, https://github.com/mainflux/mainflux/issues/1088 must be addressed quickly after this one is merged, as CT becomes redundant.

**Comment by manuio** [comment]
@drasko I removed from Nginx conf the ws-adapter endpoint. Correct me if I'm wrong but I think that MQTT over WS is configured here: https://github.com/mainflux/mainflux/blob/master/docker/nginx/nginx-key.conf#L89

I'll take #1088 when we are done with this one.

**Review by mteodor — DISMISSED** [review]

File: docker/ssl/authorization.js [hunk 1]
```diff
@@ -138,18 +138,6 @@ function setKey(r) {
...
-        for (a in r.args) {
-            if (a == 'authorization' && r.args[a] === clientKey) {
-                return clientKey;
-            }
-        }
-
-        r.error('Authorization param does not match certificate');
-        return '';
-    }
-
```
Comment (dborovcanin) [inline docker/ssl/authorization.js:152]: Please don't remove this. We want to keep MQTT over WS.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: docker/ssl/authorization.js (hunk 1 above)
Comment (manuio) [reply docker/ssl/authorization.js:152]: ok!

**Review by dborovcanin — DISMISSED** [review]

**Comment by nmarcetic** [comment]
@manuio Can you just make sure you update the vendor ? I don't see any removal regarding WS like Gorilla package. Besides this LGTM.

**Review by nmarcetic — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**